datasync/downloader/downloader.go

```diff
@@ -1359,6 +1445,13 @@ func (d *Downloader) processHeaders(origin uint64, pivot uint64, td *big.Int) er
 						}
 					}
 					// If we're importing pure headers, verify based on their recentness
+					var pivot uint64
```

Comment:
It seems no use of `pivot` variable.

Reply:
It seems the pivot is used only the following if statement.
L1456 `chunk[len(chunk)-1].Number.Uint64()+uint64(fsHeaderForceVerify) > pivot`

Reply:
got it. Thanks.